ssl/statem/statem_clnt.c

```diff
@@ -484,6 +484,8 @@ static WRITE_TRAN ossl_statem_client13_write_transition(SSL_CONNECTION *s)
             st->hand_state = TLS_ST_CW_COMP_CERT;
         else
             st->hand_state = TLS_ST_CW_CERT;
+
+        if (s->write_finished.t > 0) s->handshake_rtt = ossl_time_abs_difference(ossl_time_now(), s->write_finished);
```

Comment:
Please always put the statement on a separate line. Also it might need to be split due to long line length.